go.sum

```diff
@@ -1,6 +1,12 @@
+github.com/davecgh/go-spew v1.1.0/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
```

Comment:
Curious, where does this come from? Is this file the transitive closure of dependencies for LPMS -> m3u8 -> testify -> (others) ?

Reply:
I believe so